src/elm/Claim.elm

```diff
@@ -389,6 +387,242 @@ viewClaimCard { shared, accountName } profileSummary claim =
         ]
 
 
+viewClaimModal : LoggedIn.Model -> Model -> Bool -> Html Msg
+viewClaimModal loggedIn claim showClaimModal =
+    let
+        t =
+            loggedIn.shared.translators.t
+
+        greenTextTitleStyle =
+            "uppercase text-green text-xs"
+
+        claimVerifiersList =
+            "block mt-6 h-32 space-y-4"
+
+        photoAndTagName =
+            div []
+                [ Profile.viewLarge loggedIn.shared loggedIn.accountName claim.claimer ]
```

Comment:
After #537, `Profile.view` and `Profile.viewLarge` have been replaced by the functions in `Profile.Summary` - you need to keep track of a `Profile.Summary.Model` and have it update on your `update` function (this is that nested TEA thing you were doing on Neutro!)